policyfile.go

```diff
@@ -221,6 +221,32 @@ func (pr *PolicyFileResource) Set(ctx context.Context, acl any, etag string) err
 	return pr.do(req, nil)
 }
 
+// Set sets the [ACL] for the tailnet and returns the resulting [ACL].
+// etag is an optional value that, if supplied, will be used in the "If-Match" HTTP request header.
+func (pr *PolicyFileResource) SetAndGet(ctx context.Context, acl any, etag string) (*ACL, error) {
```

Comment:
Should we just modify the behaviour of `Set` to return the `*ACL` instead of adding a new method, or are you thinking there is worth in keeping both here to ease v1 -> v2 migration?

Reply:
I added a 2nd method for three reasons:

1. To avoid breaking clients who already use the existing method
2. To avoid doing work that the above clients don't need (i.e. deserializing the response JSON)
3. `Set` accepts both an `ACL` as well as a HuJSON string. For symmetry, that would imply that `Set` would return either an `ACL` or a HuJSON string depending on what was passed in, meaning that the return type would have to be `any`, which isn't very ergonomic.

Reply:
> Set accepts both an ACL as well as a HuJSON string

Having said that out loud, I'm now realizing that the new method does the same. Thinking ...

Reply:
Signature fixed.